grunt/githooks.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  all: {
+    'pre-commit': 'eslint',
```

Comment:
this needs to be `grunt eslint` since using just `eslint` requires eslint to be globally installed on the users machine.